### PR TITLE
feat: add audit module and support audit for clusters

### PR DESCRIPTION
**Pull request by ChenLingPeng**

### CODE REVIEW

File: cmd/tke-installer/app/installer/types/types.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ type Config struct {
 	Monitor   *Monitor  `json:"monitor,omitempty"`
 	HA        *HA       `json:"ha,omitempty"`
 	Gateway   *Gateway  `json:"gateway,omitempty"`
+	Audit     *Audit    `json:"audit"`
```
Comment (QianChenglong) [inline cmd/tke-installer/app/installer/types/types.go:41]: missing omitempty

Comment (ChenLingPeng) [reply cmd/tke-installer/app/installer/types/types.go]: resolved

**Comment by ChenLingPeng** [comment]
@choujimmy @QianChenglong  PR updated. PTAL

**Comment by ChenLingPeng** [comment]
commit rebase, squash and amend

**Pull request merged**